production.ini

```diff
@@ -411,6 +411,10 @@ session.data_dir = %(here)s/data/sessions/data
 session.lock_dir = %(here)s/data/sessions/lock
 session.key = mykey
 session.cookie_on_exception = true
+# Tell the browser to only send the cookie over TLS
+session.secure = true
```

Comment:
@pypingou We actually currently have a separate development.ini.example file that does not have this change in it, so it's sort of implicit that this change would only be for production.